safe/definitions/test/test_utilities.py

```diff
@@ -492,6 +494,22 @@ def test_set_provenance(self):
         }
         self.assertDictEqual(provenance_collection, expected)
 
+    def test_get_provenance(self):
+        """Test fot get_provenance"""
```

Comment:
Full stop is better ;-)
But I don't mind for test file

Reply:
Thanks :D